docs/appendices/release-notes/unreleased.rst

```diff
@@ -52,6 +52,9 @@ None
 Changes
 =======
 
+- Introduced new optional ``RETURNING`` clause for :ref:`Update <ref-update>` to
+  return values based on each row actually updated.
```

Comment:
```suggestion
  return specified values from each row updated.
```

Reply:
based on my understanding of your previous comment. lemme know if this still isn't accurate!

Reply:
👍

Reply:
👍